stardis/model/geometry/radial1d.py

```diff
@@ -1,5 +1,7 @@
-class Radial1DGeometry:
+from tardis.io.util import HDFWriterMixin
```

Comment:
I don't think we need this import anymore since Radial1DGeometry won't need the HDF writer

Reply:
Addressed in https://github.com/tardis-sn/stardis/pull/213/commits/140f06b511f1555f4c6f96f469126b24ff35a344